javascript/javascript1/week3/smartPhone.js

```diff
@@ -0,0 +1,48 @@
+// Adding an activity
+
+
+ const activities=[]; 
+function addActivity( date, activity, duration){
+    
+    activities.push({date, activity, duration});
+ return activities;
+}
+addActivity("23/7-18", "YouTube",30);
+addActivity("24/7-18","Facebook", 40);
+addActivity("15/7-18", "twitter", 25); 
+addActivity("15/7-18" , "Instegram", 18); 
+console.log(activities); 
+
+// Show my status: 
+
+function showStatus(activities){
+    let total=0; 
+      for(let i = 0; i<= activities.length; i++){
+        total += activities[i].duration; 
+        if(! i === ""){
+
+            console.log(` you hav added 4 activities ${activities[i].activity}. They amount to ${total} min.of usage`);
+
+        }
+ else return `Add some activities before calling showStatus`; 
+      }
+     
+}
+console.log(showStatus(activities)); 
+
+
+// Usage limit: 
+ //function 
+
+ function usageLimit(storageLimit){
+    if( storageLimit==="number" && storageLimit<= showStatus(activities)){
```

Comment:
+1 for approach: it's very good to use already existing function.
But does it work? 
Let's think:
showStatus method returns `undefined` always. 
this means that we are doing checks
```
storageLimit <= undefined
```
and that condition evaluates to `false` always.